iaac/terraform/kubeflow-components/cluster-local-gateway/main.tf

```diff
@@ -0,0 +1,15 @@
+module "helm_addon" {
+  source            = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons/helm-addon?ref=v4.9.0"
+  helm_config       = local.helm_config
+  addon_context     = var.addon_context
+}
+
+# Wait 60 seconds after helm chart deployment
+resource "time_sleep" "wait_60_seconds" {
```

Comment:
definitely need to have deterministic install

Reply:
adding todo

Reply:
done